README.md

```diff
@@ -60,6 +60,14 @@ dhapi -h
 dhapi buy_lotto645 -q # 프로필 파일을 이용해 계정 정보 입력 & 자동모드로 5장 구매
 ```
 
+### 스케줄
+
+```sh
+crontab -e
+
+0 0,6-23 * * * PATH=$PATH:/opt/homebrew/bin && dhapi buy_lotto645 -q
```

Comment:
What does `0 0,6-23 * * *` mean? This cronjob will fail every 125 of 126. I think the cron schedule should be like `0 0 * * 1` or `0 0 * * FRI`. 

If you're assuming an unusual request failure (e.g., network connection shortage), it would be better to notify it somehow.

Reply:
```
로또 6/45의 구입안내
매일 6시부터 24시까지 1년 365일 연중무휴 판매합니다. 추첨일(토요일)에는 오후 8시에 판매 마감합니다. 추첨일 오후 8시부터 다음날(일요일) 오전 6시까지는 판매가 정지됩니다.
```

https://crontab.guru/#0_0,6-23_*_*_*
<img width="1406" alt="스크린샷 2023-06-06 오후 4 01 57" src="https://github.com/roeniss/dhlottery-api/assets/8468371/ba7809b3-a920-472f-acac-cd6f4e032cc6">

see this~~!

Reply:
@seunggabi In dhlottery online page, you can only buy 5 tickets per week. Have you tested your crontab to see what happen?

Reply:
이거 늦게 이해했는데, 
맥북인 경우, 컴퓨터가 언제 켜질지 몰라서, 매시간마다 호출하는게 좋아보였어요.
말씀하신 것처럼 매주에 처음 한번만 정상구매가 일어나긴한데, 언제 컴퓨터를 켤지 모르니..ㅠ

Reply:
@seunggabi I'm happy that you get my idea! 

> 언제 컴퓨터를 켤지 모르니..ㅠ

You are assuming that you're gonna boot your MacBook up once a week at least and that's highly unstable. So your cron still doesn't cover every corner case (e.g., you bootup after 8 day from last boot)

> 컴퓨터가 언제 켜질지 몰라서

In this case, I suggest you check out this issue:
- https://github.com/roeniss/dhlottery-api/issues/12